workflow-ui/compose/src/main/java/com/squareup/workflow1/ui/compose/ComposedViewFactory.kt

```diff
@@ -0,0 +1,141 @@
+// See https://youtrack.jetbrains.com/issue/KT-31734
+@file:Suppress("RemoveEmptyParenthesesFromAnnotationEntry")
+
+package com.squareup.workflow1.ui.compose
+
+import android.content.Context
+import android.view.View
+import android.view.ViewGroup
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.platform.ComposeView
+import com.squareup.workflow1.ui.LayoutRunner
+import com.squareup.workflow1.ui.ViewEnvironment
+import com.squareup.workflow1.ui.ViewFactory
+import com.squareup.workflow1.ui.WorkflowUiExperimentalApi
+import com.squareup.workflow1.ui.bindShowRendering
+import kotlin.reflect.KClass
+
+/**
+ * Creates a [ViewFactory] that uses a [Composable] function to display the rendering.
+ *
+ * Simple usage:
+ *
+ * ```
+ * val FooViewFactory = composedViewFactory { rendering, _ ->
```

Comment:
Just double-checking that the capitalization of `FooViewFactory` was deliberate... (same below).

Reply:
yep